client-app/src/examples/contact/TileView.js

```diff
@@ -0,0 +1,39 @@
+import {hoistCmp} from '@xh/hoist/core';
+import {div, tileFrame, vframe} from '@xh/hoist/cmp/layout';
+import {Icon} from '@xh/hoist/icon/Icon';
+import './TileView.scss';
+
+export const tileView = hoistCmp.factory({
+    render({model}) {
+        const {store, selModel} = model.gridModel;
+
+        return tileFrame({
+            spacing: 10,
+            desiredRatio: 1.25,
+            minTileWidth: 300,
+            maxTileWidth: 600,
+            minTileHeight: 250,
+            maxTileHeight: 250,
+            items: store.records.map(record => {
+                const isSelected = selModel.selectedRecordId === record.id;
```

Comment:
No difference in functionality, but you could destructure `{selectedRecord}` off of the model at the top and then compare each record to it directly.